libs/gltfio/src/ResourceLoader.cpp

```diff
@@ -302,20 +305,51 @@ ResourceLoader::~ResourceLoader() {
 }
 
 void ResourceLoader::addResourceData(const char* uri, BufferDescriptor&& buffer) {
+    pImpl->addResourceData(uri, std::move(buffer));
+}
+
+static bool endsWith(std::string_view expr, std::string_view ending) {
+    if (expr.length() >= ending.length()) {
+        return (expr.compare(expr.length() - ending.length(), ending.length(), ending) == 0);
+    }
+    return false;
+}
+
+// TODO: This is not a great way to determine if a resource is a texture, but we can remove it after
+// gltfio gains support for concurrent downloading of vertex data:
+// https://github.com/google/filament/issues/5909
+static bool isTexture(const char* uri) {
```

Comment:
What about .ktx?

Reply:
Gah, excellent catch, we support ktx2 containers in glTF so I need to check for that extension.  Thanks!  (hopefully will get rid of these checks entirely before too long)